src/api/movies.js

```diff
@@ -2,9 +2,7 @@ import { axiosInstance } from './config';
 
 export default {
   get(params) {
-    return axiosInstance.get('/movies', {
-      params,
-    });
+    return axiosInstance.get(`/movies/?${params}`);
```

Comment:
It's not right, previous version was correct. We should add `paramSerializer` to axios config ([here](https://github.com/IlonaMenkui/super-enigma-fe/blob/develop/src/api/config.js#L5)) as follows:
```js
paramsSerializer: (params) => {
    return queryString.stringify(params);
  },
```